src/tlo/methods/breast_cancer.py

```diff
@@ -665,45 +680,53 @@ def apply(self, person_id, squeeze_factor):
 
         df.at[person_id, 'brc_breast_lump_discernible_investigated'] = True
 
-        # Use a biopsy to diagnose whether the person has breast Cancer:
-        # todo: request consumables needed for this
+        # Check consumables to undertake biopsy are available
+        cons_avail = self.get_consumables(item_codes=self.module.item_codes_breast_can['screening_biopsy_core'],
+                                          optional_item_codes=
+                                          self.module.item_codes_breast_can['screening_biopsy_optional'])
 
-        dx_result = hs.dx_manager.run_dx_test(
-            dx_tests_to_run='biopsy_for_breast_cancer_given_breast_lump_discernible',
-            hsi_event=self
-        )
+        if cons_avail:
```

Comment:
Shouldn't be the hsi rescheduled if consumables not available?

Reply:
I think this would be a question from @andrew-phillips-1 as we would need to think on if this would happen in practice, at what time the individual would be asked to return, what happens if the consumables arent available again... 

Maybe its ok to leave for now and mark as an assumption

Reply:
I think the consumable availability condition might have been added since I first developed this module as I can't recall considering this issue.  I suggest leaving as it is for now - hopefully at some point I or someone else will get a chance to dig into what would actually happen in these circumstances.